robin_stringer/gulpfile.js

```diff
@@ -0,0 +1,45 @@
+var gulp = require('gulp');
+var webpack = require('webpack-stream');
+var minifyCss = require('gulp-minify-css');
+var gulpWatch = require('gulp-watch');
+var sass = require('gulp-sass');
+var maps = require('gulp-sourcemaps');
+
+gulp.task('static:dev', function() {
+  gulp.src('app/**/*.html')
+  .pipe(gulp.dest('build/'));
+});
+
+gulp.task('sass:dev', function() {
+  gulp.src('./app/sass/**/*.scss')
+  .pipe(sass().on('error', sass.logError))
+  .pipe(gulp.dest('build/'));
+});
```

Comment:
I don't see the maps and minification still.

gulp.task('sass:dev', function() {
  gulp.src('./app/sass/**/*.scss')
  .pipe(maps.init())
  .pipe(sass().on('error', sass.logError))
  .pipe(minifyCss())
  .pipe(maps.write('./'))
  .pipe(gulp.dest('build/'));
});